modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -183,12 +183,16 @@ def preprocess_func(cls, map_func):
     # END Abstract Methods
 
     @classmethod
-    def create_partition_from_metadata(cls, **metadata):
+    def create_partition_from_metadata(
+        cls, dtypes: Optional[pandas.Series] = None, **metadata
+    ):
         """
         Create NumPy array of partitions that holds an empty dataframe with given metadata.
 
         Parameters
         ----------
+        dtypes : pandas.Series, optional
+            Dtypes that will be used when calling `astype`.
```

Comment:
```suggestion
            Column dtypes.
            Upon creating a pandas DataFrame from `metadata` we call `astype` since
            pandas doesn't allow to pass a list of dtypes directly in the constructor.
```